Content.Shared/Actions/BaseActionComponent.cs

```diff
@@ -120,6 +120,12 @@ public abstract partial class BaseActionComponent : Component
     /// </summary>
     [ViewVariables] public EntityUid? AttachedEntity;
 
+    /// <summary>
+    ///     The preferred entity to raise an action event on.
+    /// </summary>
+    [DataField("raiseOnUser"), ViewVariables]
```

Comment:
```suggestion
    [DataField]
```
DataField implies VV, and the field name is now optional